### ISSUE TITLE
Simplified setup for Batarangle using ELEMENT_PROBE_PROVIDERS

### ISSUE THREAD

**Issue by juristr** [rangle/batarangle#126]
Your current setup suggests the following to make debugging with Batarangle work.

```javascript
import {bootstrap} from 'angular2/platform/browser';
...

// For Batarangle Integration
import {AppViewListener} from 'angular2/src/core/linker/view_listener';
import {DebugElementViewListener, inspectNativeElement}
  from 'angular2/platform/common_dom';
import {bind} from 'angular2/core';

...

bootstrap(App, [
        ...HTTP_PROVIDERS,
        ...ROUTER_PROVIDERS,
        provide(LocationStrategy, { useClass: HashLocationStrategy }),
        bind(AppViewListener).toClass(DebugElementViewListener)
    ]);
```

However, I found that this setup should be enough.

```javascript
import {bootstrap, ELEMENT_PROBE_PROVIDERS} from 'angular2/platform/browser';
...

bootstrap(App, [
        ...
        ...ELEMENT_PROBE_PROVIDERS
    ]);
```

It seems to work on my local setup. If u wish I can send a PR to update the README.md

**Comment by sumitarora** [comment]
@juristr With updated beta.3 release of Angular, this configuration is no longer needed. We will update the readme with the support of beta.3.

**Comment by juristr** [comment]
@sumitarora You mean the `ELEMENT_PROBE_ROVIDERS`? Could you provide me a link to more info about that? I'm demoing Batarangle as part of a screencast series on Angular 2 so I'd like to make sure to do the right setup.

Thx a lot!

**Comment by sumitarora** [comment]
@juristr As a part on beta.3 if the application is being run in dev mode we don't need to do any configuration. `ng.probe` is available on the window itself and can be used to debug elements. 

Batarangle currently supports application written using Angular beta.2 and steps to configure the application for that are same as those written in the readme.

**Comment by juristr** [comment]
Solved with #130
